composer.json

```diff
@@ -28,6 +28,7 @@
     "require":
     {
         "php":                  "^5.5|7.0",
+        "ext-memcache":         "*",
```

Comment:
Latest release is from 2013 so will do that for now:
https://pecl.php.net/package/memcache